### ISSUE TITLE
Desktop menu bar isn't working on macos

### ISSUE THREAD

**Issue by lovelypeach** [fyne-io/fyne#1905]
### Describe the bug:

The desktop menu bar is not clickable after app is ran. But when I switch to another app and switch back, the desktop menu bar can be clicked. Even if the simpest fyne app. And fyne_demo is alike.

### Example code:

```go
func main() {
	a := app.New()
	w := a.NewWindow("Test")
	w.SetContent(container.NewWithoutLayout())
	w.Resize(fyne.NewSize(300, 300))
	w.ShowAndRun()
}
```

### Device (please complete the following information):
 - **OS:** MacOS Catalina 
 - **Version:** 10.15.7 
 - **Go version:** go1.15.6 darwin/amd64 
 - **Fyne version:** v2.0.0

**Comment by andydotxyz** [comment]
This was fixed for 2.0.0, are you certain that you are using the latest version?
Until the fix is pushed upstream you will need a replace directive in your app go.mod

`replace github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200625191551-73d3c3675aa3 => github.com/fyne-io/glfw/v3.3/glfw v0.0.0-20201123143003-f2279069162d`

in an updated version the need for this will go away

**Comment by lovelypeach** [comment]
It works after replaced glfw. Thank you very much.
